pkg/rke/k8s_version_info.go

```diff
@@ -667,6 +667,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.7.5-patch0",
 			MinRKEVersion:     "1.4.0-rc0",
 		},
+		"1.26.13-rancher1-1": {
```

Comment:
We already have min versions set for v1.26, so don't need to set it for any v1.26.x-rancher versions if they match with `MinRancherVersion`

Reply:
removed the code block.